code/game/objects/items/toys.dm

```diff
@@ -211,6 +211,22 @@
 	attack_verb = list("attacked", "struck", "hit")
 	brightness_on = 0
 
+/obj/item/weapon/twohanded/dualsaber/toy/attack(target as mob, mob/living/user as mob)
+	if(HULK in user.mutations)
+		to_chat(user, "<span class='warning'>You grip the blade too hard and accidentally close it!</span>")
+		unwield()
+		return
+	..()
+	if((CLUMSY in user.mutations) && (wielded) &&prob(40))
+		to_chat(user, "<span class='warning'>You twirl around a bit before losing your balance and impaling yourself on the [src].</span>")
+		unwield()

```

Comment:
I could just remove this whole section but I want to settle the parent issue above first